sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2596,21 +2589,21 @@
       "type": "object",
       "properties": {
         "jobId": {
-          "description": "System generated.  Ignored at creation.",
+          "description": "The unique identifier of the job.",
           "type": "string"
         },
         "startTimeUtc": {
           "format": "date-time",
-          "description": "System generated.  Ignored at creation.",
+          "description": "The start time of the job in UTC.",
           "type": "string"
         },
         "endTimeUtc": {
           "format": "date-time",
-          "description": "System generated.  Ignored at creation.\r\nRepresents the time the job stopped processing.",
+          "description": "Represents the time the job stopped processing.",
```

Comment:
Just "The time the job stopped processing" should be fine

Reply:
In general, I see a few places in the swagger where "Represents is used". Do you all think we should remove that as the starting word whenever we see it for language consistency?

Reply:
I looked at the OpenAPI specifications. The examples they have in the documentation do not include a 'Represents' or something of that sort. I opt for removing any 'Represents' from the document.